packages/block-library/src/credits/block.json

```diff
@@ -0,0 +1,25 @@
+{
+	"apiVersion": 2,
+	"name": "core/credits",
+	"title": "Credits",
+	"category": "design",
+	"description": "Displays credits for the site.",
+	"textdomain": "default",
+	"attributes": {
+		"contentID": {
+			"type": "number",
+			"default": 0
+		},
+		"content": {
+			"type": "string",
+			"default": ""
+		},
+		"textAlign": {
+			"type": "string"
+		}
+	},
```

Comment:
If we do decide to show it in the inserter we might want to add some keywords:
`credit` and `wordpress` would make sense.